azureml_deployment/src/main.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+import pandas as pd
+import joblib
+from fastapi.responses import JSONResponse
+
+app = FastAPI()
+
+# Pydantic models for new data points.
+class TestDatapoint(BaseModel):
+    x1: float = None
+    x2: float = None
+
+
+class PredictionRequest(BaseModel):
+    data: List[TestDatapoint] = []
+
+
+# Load the model
+with open("models/model.joblib", "rb") as model_pickle:
```

Comment:
Please add an explanation why we load it outside of the function instead of inside the function.
THis is important to know